code/modules/mob/living/carbon/xenomorph/castes/widow/abilities_widow.dm

```diff
@@ -166,6 +166,43 @@
 	spiderlings -= source
 	UnregisterSignal(source, list(COMSIG_MOB_DEATH, COMSIG_PARENT_QDELETING))
 
+// ***************************************
+// *********** Spiderling mark
+// ***************************************
+
+/datum/action/xeno_action/activable/spiderling_mark
+	name = "Spiderling Mark"
+	ability_name = "spiderling_mark"
+	desc = "Send your spawn on a human"
+	action_icon_state = "hunter_mark"
+	plasma_cost = 50
+	cooldown_timer = 5 SECONDS
+	keybinding_signals = list(
+		COMSIG_XENOABILITY_SPIDERLING_MARK,
+	)
+
+/datum/action/xeno_action/activable/spiderling_mark/use_ability(atom/A)
+	. = ..()
+	if(!ishuman(A))
+		owner.balloon_alert(owner, "Not a human")
+		return fail_activate()
+	var/mob/living/carbon/human/victim = A
+	if(victim.stat != CONSCIOUS)
+		owner.balloon_alert(owner, "Already down")
+		return fail_activate()
+	var/datum/action/xeno_action/create_spiderling/create_spiderling_action = owner.actions_by_path[/datum/action/xeno_action/create_spiderling]
+	if(isnull(create_spiderling_action.spiderlings))
+		owner.balloon_alert(owner, "No spiderlings")
+		return fail_activate()
+	/// Here we make every single spiderling that we have also burrow and assign a signal so that they unburrow too
```

Comment:
this shouldn't be autodoc, also you're not actually sending a signal so I wouldn't use that wording.